src/main/java/com/onfleet/demo/homework/TaskAssigner.java

```diff
@@ -0,0 +1,62 @@
+package com.onfleet.demo.homework;
+
+
+import com.onfleet.demo.homework.collection.Tasklist;
+import com.onfleet.demo.homework.manager.BlindTaskManager;
+import com.onfleet.demo.homework.manager.TaskManager;
+import com.onfleet.demo.homework.struct.Driver;
+import com.onfleet.demo.homework.struct.Task;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.LinkedHashSet;
+import java.util.Map;
+
+
+/**
+ * Specifies a common interface between {@link TaskManager} and {@link BlindTaskManager},
+ * where {@link Task} records may be assigned to {@link Driver} {@link Tasklist}s.
+ */
+public interface TaskAssigner {
```

Comment:
general `TaskAssigner` interface, which presents an API for using `TaskManager` and `BlindTaskManager` agnostically